ofl/dancingscript/OFL.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 2016 The Dancing Script Project Authors (impallari@gmail.com), with Reserved Font Name 'Dancing Script’.
+Copyright 2016 The Dancing Script Project Authors (https://github.com/impallari/DancingScript), with Reserved Font Name 'Dancing Script'.
```

Comment:
Looks like the TN repo is the new source. Can you update this copyright string, or even better send impallari a pull request. I don't know if he'll merge it but it's worth a shot.

Reply:
It might make sense to move this one over to a `googlefonts` repro. I'd flagged it in my sheet for Dave, but I'll email him directly about it.